lib/Ohmbrewer_Menu.cpp

```diff
@@ -0,0 +1,156 @@
+#include "Ohmbrewer_Menu.h"
```

Comment:
Wee can make a subdirectory to house all the menu files, and that will consolidate them and not clutter the main folder while at the same time allowing us to maintain the same conventions we have been using thus far.